pylint/checkers/classes/special_methods_checker.py

```diff
@@ -208,7 +219,7 @@ def _check_unexpected_method_signature(self, node):
             # to take all of them in consideration.
             emit = mandatory not in expected_params
             # pylint: disable-next=consider-using-f-string
-            expected_params = "between %d or %d" % expected_params
+            expected_params = "between %d or %d" % expected_params  # type: ignore[assignment]
```

Comment:
?

Reply:
mypy has a hard time with the length of the expected_params tuple.

Reply:
If we use an f-string does the message go away?

Reply:
No, mypy do not catch the isinstance( ... tuple) just above. But at least we would not crash for longer tuples.